### ISSUE TITLE
Upgrade to webpack 4

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#4908]
This issue is to track updating to webpack 4. The update is likely to involve a few dependencies. 

tofumatt had a work in progress PR here: https://github.com/mozilla/addons-frontend/pull/4697 it might be useful to use that as a starting point. 

### QA notes:

Once this is marked fixed we'll need to double check the site is still working as expected since this could potentially impact our JS. Tests should provide pretty good coverage but it will still be worth looking out for issues.

**Comment by willdurand** [comment]
Once we have webpack 4, we can upgrade `deepcopy`, see: https://github.com/mozilla/addons-frontend/pull/5054.

**Comment by willdurand** [comment]
We also want to upgrade the loaders:

- CSS loader: css-loader

**Comment by willdurand** [comment]
Updated the list of things to check for QA and asked for extra QA since this is criticial.

**Comment by ioanarusiczki** [comment]
@willdurand 

- HCT (telemetry) did not change since last week which means I could test it only on AMO dev with FF65 and using for Disco Pane this URL https://discovery.addons-dev.allizom.org/en-US/firefox/discovery/pane/57.0/Darwin/normal.  It won't work from about:addons. There's a Console message [GA: ON] Not logging to telemetry since hctEnabled is disabled. 

I did not receive events with FF63 as it's discussed here https://github.com/mozilla/addons-frontend/issues/6952 

-since HCT is still under testing in -dev I don't think it's a problem. What's more important is AMO's and disco pane's functionalities to be tested so I verified AMO, Disco Pane and installation with FF63 (release &unbranded on Win10) but the final validation will be made after the release testing is complete tomorrow.
